include/lemlib/logger/telemetrySink.hpp

```diff
@@ -4,14 +4,24 @@
 
 namespace lemlib {
 /**
- * @brief A sink where telemetry data goes
+ * @brief Sink for sending telemetry data.
  *
+ * This is the primary way of interacting with the telemetry portion of LemLib's logging implementation. This sink is
+ * used for sending data that is not meant to be viewed by the user, but will still be used by something else, like a
+ * data visualization tool. Messages sent through this sink will not be cleared from the terminal and not be visible to
+ * the user.
+
+ * <h3> Example Usage </h3>
+ * @code
+ * #define TEMPERATURE_ID 1
+ * lemlib::telemetrySink()->setLowestLevel(lemlib::Level::INFO);
+ * lemlib::telemetrySink()->info("{},{},{}", TEMPERATURE_ID, motor1.get_temperature(), motor2.get_temperature());
```

Comment:
```suggestion
 * lemlib::telemetrySink()->setLowestLevel(lemlib::Level::INFO);
 * lemlib::telemetrySink()->info("{},{}", motor1.get_temperature(), motor2.get_temperature());
```
I don't really think that the id is necessary in this example.